rds.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_iam_policy_document" "rds_for_github" {
+
+  statement {
+    sid    = "AllowListDescribe"
+    effect = "Allow"
+    actions = [
+      "rds:Describe*",
```

Comment:
- for `DescribeAccountAttributes`, the output as described in https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeAccountAttributes.html looks like it would actually be useful for teams, ourselves included ;)
 - for `DescribeCertificates`, we don't use any so n/a